x/sequencer/types/params.go

```diff
@@ -47,19 +52,20 @@ func DefaultParams() Params {
 	}
 	minBond := sdk.NewCoin(denom, sdk.NewIntFromUint64(DefaultMinBond))
 	return NewParams(
-		minBond, DefaultUnbondingTime,
+		minBond, DefaultUnbondingTime, DefaultNoticePeriod,
 	)
 }
 
 // ParamSetPairs get the params.ParamSet
 func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
 	return paramtypes.ParamSetPairs{
 		paramtypes.NewParamSetPair(KeyMinBond, &p.MinBond, validateMinBond),
-		paramtypes.NewParamSetPair(KeyUnbondingTime, &p.UnbondingTime, validateUnbondingTime),
+		paramtypes.NewParamSetPair(KeyUnbondingTime, &p.UnbondingTime, validateTime),
+		paramtypes.NewParamSetPair(KeyNoticePeriod, &p.NoticePeriod, validateTime),
 	}
 }
 
-func validateUnbondingTime(i interface{}) error {
```

Comment:
we should [validate unbonding time > grace period time](https://github.com/dymensionxyz/dymension/issues/613). Also why are we using time here vs blocks (like in the dispute period case?)
seems like it doesn't account for hub going down scenarios. 

in that case, if you have grace period in blocks, and unbonding time in time, it's not correlated.